Cannot run CLLI - Error:'MODULE_NOT_FOUND'

**Describe the bug**
Hi,

I tried to run the latest version (v1.10.0) and receive the following error:
`./dcli-linux -h
pkg/prelude/bootstrap.js:1872
      throw error;
      ^

Error: Cannot find module '/snapshot/dashlane-cli/node_modules/@json2csv/plainjs/dist/cjs/index.js'
1) If you want to compile the package/file into executable, please pay attention to compilation warnings and specify a literal in 'require' call. 2) If you don't want to compile the package/file into executable and want to 'require' it from filesystem (likely plugin), specify an absolute path in 'require' call using process.cwd() or process.execPath.
    at createEsmNotFoundErr (node:internal/modules/cjs/loader:960:15)
    at finalizeEsmResolution (node:internal/modules/cjs/loader:953:15)
    at resolveExports (node:internal/modules/cjs/loader:482:14)
    at Function.Module._findPath (node:internal/modules/cjs/loader:522:31)
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:919:27)
    at Function._resolveFilename (pkg/prelude/bootstrap.js:1951:46)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at Module.require (pkg/prelude/bootstrap.js:1851:31)
    at require (node:internal/modules/cjs/helpers:102:18) {
  code: 'MODULE_NOT_FOUND',
  path: '/snapshot/dashlane-cli/node_modules/@json2csv/plainjs/package.json',
  pkg: true
}`

**To Reproduce**
This happens in Windows and Linux with version v1.10.0. 
In both cases i tried to run the "-h" command.
"dcli-win-x64.exe -h" with the prebuild from Github and for Linux with the yarn build I did on my own and then run "./dcli-linux -h"

**Expected behavior**
Running the "-h" command and returning the menu options.

**Environment (please complete the following information):**
 - OS: Windows11 and Linux Docker Image:"node:16.20.1"
 - Version v1.10.0

--- a/package.json
+++ b/package.json
@@ -9,7 +9,11 @@
     },
     "pkg": {
         "assets": [
-            "node_modules/better-sqlite3/build/Release/better_sqlite3.node"
+            "node_modules/better-sqlite3/build/Release/better_sqlite3.node",
+            "node_modules/@json2csv/plainjs/dist/**/*",
+            "node_modules/@json2csv/formatters/dist/**/*",
+            "node_modules/@json2csv/transforms/dist/**/*",
+            "node_modules/@streamparser/json/dist/**/*"
         ]
     },
     "scripts": {

